test/node/features/fixtures/express/scenarios/app.js

```diff
@@ -70,6 +104,26 @@ app.get('/throw-non-error', function (req, res, next) {
   throw 1 // eslint-disable-line
 })
 
+app.get('/oversized', function (req, res, next) {
+  function repeat(s, n){
+    var a = [];
+    while(a.length < n){
+        a.push(s);
+    }
+    return a.join('');
+  }
+
+  var big = {};
+  var i = 0;
+  while (JSON.stringify(big).length < 5*10e5) {
+    big['entry'+i] = repeat('long repetitive string', 1000);
+    i++;
+  }
+  req.bugsnag.leaveBreadcrumb('big thing', big);
```

Comment:
works even though apparently not supported on node because any cloned clients don't get the patched method

Reply:
Yeah I noticed this with another monkey patched method recently — I think it's actually a good thing in this case though, since the breadcrumbs would be scoped to the request client so it works as expected?

Reply:
Yes, it works as you would expect. I have manually tested it for a different purpose.